src/utils/Utils.tsx

```diff
@@ -882,4 +890,49 @@ export default class Utils {
   private static getDeviceLanguage(): string {
     return Utils.getLanguageFromLocale(Utils.getDeviceLocale());
   }
+
+  public static markerStatusChargingStations(connectorTab: Connector[], inactive: boolean) {
+    if (inactive) {
+      return charging_unavailable;
+    } else if (connectorTab.find((connector) => connector.status === ChargePointStatus.AVAILABLE)) {
+      return charging_available;
+    } else if (
+      connectorTab.find((connector) => connector.status === ChargePointStatus.FINISHING) ||
+      connectorTab.find((connector) => connector.status === ChargePointStatus.PREPARING)
+    ) {
+      return charging_finishing;
```

Comment:
Maybe give a name for both preparing and finishing to the marker image

Reply:
Do you want me to change the name of the image?